src/condition/scalar_condition.cpp

```diff
@@ -65,7 +64,8 @@ ResultType eval_object(Iterator &it, std::string_view address, bool ephemeral,
                 if constexpr (std::is_same_v<ResultType, bool>) {
                     return true;
                 } else {
-                    return {{{{"input"sv, object_to_string(dst), address, it.get_current_path()}},
+                    return {{{{"input"sv, object_view{dst}.convert<std::string>(), address,
```

Comment:
this stuff could be directly on `owned_object` too

Reply:
Addressed in https://github.com/DataDog/libddwaf/pull/387/commits/95ba3b1f3430ba12ec3061a583669015d0d98deb